sql-cli/sql_cli/project.py

```diff
@@ -41,10 +58,15 @@ def _update_config(self) -> None:
         :param airflow_dags_folder: Custom user-defined Airflow DAGs folder
         """
         config = Config(environment=DEFAULT_ENVIRONMENT, project_dir=self.directory)
-        if self.airflow_home is not None:
-            config.write_value_to_yaml("airflow", "home", str(self.airflow_home))
-        if self.airflow_dags_folder is not None:
-            config.write_value_to_yaml("airflow", "dags_folder", str(self.airflow_dags_folder))
+        if self._airflow_home is not None:
+            config.write_value_to_yaml("airflow", "home", str(self._airflow_home))
+        if self._airflow_dags_folder is not None:
+            config.write_value_to_yaml("airflow", "dags_folder", str(self._airflow_dags_folder))
+
+    def _initialise_airflow(self) -> None:
+        cmd = f'PYTHONWARNINGS="ignore" AIRFLOW__CORE__LOAD_EXAMPLES=False AIRFLOW_HOME={self.airflow_home} airflow db init'  # noqa: E501
+        os.system(cmd)
+        # replace by subprocess.run
```

Comment:
Because we have airflow as a python dependency we could also just call the [python interface](https://github.com/apache/airflow/blob/main/airflow/cli/commands/db_command.py#L34-L38) for initialising the db. I think this is cleaner.

Reply:
Note that Airflow’s Python interface has a ton of side effects (we’re working on reducing them but it’s still not all the way there). It’s probably still a good idea to use it here for now to see how far we can go.